packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -6,6 +6,8 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:usage/src/uuid.dart'; // ignore: implementation_imports
```

Comment:
I agree with Adam that it would be cleaner if the package exported the needed API.